Softeq.NetKit.Components.EventBus.Service/Connection/IServiceBusPersisterConnection.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// Developed by Softeq Development Corporation
 // http://www.softeq.com
 
+using System;
 using Microsoft.Azure.ServiceBus.Core;
 
 namespace Softeq.NetKit.Components.EventBus.Service.Connection
 {
-    public interface IServiceBusPersisterConnection
+    public interface IServiceBusPersisterConnection : IAsyncDisposable
```

Comment:
Does it make sense to implement IDisposable as well? Just to avoid leaks if some consumer uses DIspose() only.

Reply:
I'm not sure how actually we can do that. The underhood connections have only async closing methods, so probably it makes sense to implement only the async Dispose and force the user to dispose asynchronously, as Microsoft.Azure.ServiceBus actually does.